package.json

```diff
@@ -31,12 +31,15 @@
     "svelte": "^3.44.0"
   },
   "devDependencies": {
-    "@sveltejs/adapter-auto": "next",
+    "@sveltejs/adapter-vercel": "next",
```

Comment:
Vercel adapter no longer required to deploy to Vercel. Adapter auto handles it under the current beta. All that's needed is the Vercel CLI to deploy run `vercel`.